src/mcpserver.ts

```
@@ -134,5 +140,246 @@ server.registerTool(
+        }
+    },
+);
+
+server.registerTool(
+    "clear_done_shoppinglist",
+    {
+        title: "Clear all done shoppinglist entries",
+        description: "Delete all Entries from the users shoppinglist where isDone = true. If you want to give a summary of the remaining shopping list call tool get_shoppinglist_entries",
+        inputSchema: {}
+    },
+    async ({ }) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)